common/types/provider.go

```diff
@@ -22,13 +22,16 @@ import (
 	"github.com/google/cel-go/common/types/pb"
```

Comment:
Rather than add these functions into the `provider.go` consider moving them into the `cel` package close to the code that translates between `CheckeExpr` and `Ast`.

Reply:
which file would that be?

Reply:
Try `cel/io.go` as the file to add these functions. Don't forget to add tests in `io_test.go`